support/rally.js

```diff
@@ -191,13 +190,13 @@ export class Rally {
    * @throws {Error} if either the key id or the JWK key object are
    *         invalid.
    */
-  _validateEncryptionKey(keyId, key) {
-    if (typeof keyId !== "string") {
-      throw new Error(`Rally._validateEncryptionKey - Invalid encryption key id ${keyId}`);
+  _validateEncryptionKey(key) {
+    if (typeof key !== "object") {
+      throw new Error("Rally._validateEncryptionKey - Invalid encryption key", key);
```

Comment:
I changed this because otherwise this gets converted to `[object Object]`, not super helpful :)